protocol-units/da/m1/light-node-verifier/src/permissioned_signers/mod.rs

```diff
@@ -45,7 +45,7 @@ where
 	pub fn new(
 		celestia_client: Arc<Client>,
 		celestia_namespace: Namespace,
-		known_signers_sec1_bytes: HashSet<Vec<u8>>,
+		known_signers_sec1_bytes: HashSet<String>,
 	) -> Self {
```

Comment:
This could be more generic to allow passing in e.g. an array of string literals:

```rust
	pub fn new<T>(
		celestia_client: Arc<Client>,
		celestia_namespace: Namespace,
		known_signers_sec1_bytes: T,
	) -> Self
        where
            T: IntoIterator,
            T::Item: Into<String>,
        {
```

Reply:
Will update.